install.yml

```diff
@@ -5,7 +5,7 @@
 
 - hosts: tester
   vars_files:
-    - vars/{{ ((test|default('')).ansible|default('')).vars|basename|default('install.yml')}}
```

Comment:
I think it is best practice to use jinja2 templates to default.

Reply:
Also, this is to default the value when the cli args are not present...

Reply:
My idea was to point to the variable file, and if a user wanted to, they can make changes to the vars file itself. If the user want to pass a new file, they'll still have to know about the default file to begin with and then update their copy, rather we can simplify things and just let user edit the default file.

Reply:
I don't quite think that approach is quite right.

Reply:
Okay tbh the approach was used because it worked, as when I ran infrared install using

```
infrared browbeat --workloads rally --install yes --config plugins/browbeat/browbeat-config.yaml -e @plugins/browbeat/all.yml
```

[This error](https://pastebin.com/PH4ZGxjP) would pop up, and I wasn't sure what the problem was wrt jinja2 templating.

Reply:
LGTM.